bats/sql-add.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+
+    dolt sql <<SQL
+CREATE TABLE test (
+    pk int primary key
+);
+CREATE TABLE test2 (
+    pk int primary key
+);
+INSERT INTO test VALUES (0),(1),(2);
+SQL
```

Comment:
Why two different SQL blocks?

Reply:
To test addition with different table name

Reply:
I mean that you had two `dolt sql <<SQL` commands, when one would do. You've fixed that now.